posts/2022-01-24-whats-in-a-boostagram.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "What's in a Boostagram"
+summary: 'Closing the Loop is a podcast about Bitcoin and the entrepreneurs, developers, and thinkers who are pushing its evolution. But on top of that, it is also a playground for us to explore and experiment with the technological and financial possibilities that Bitcoin brings to online publishing.'
+date: '2022-01-24'
+blocktime: '720166'
+author: '@dnlggr'
+linkAuthorTwitter: true
+thumbnail: 'thumbnail.jpeg'
+cover: 'cover.jpeg'
+coverImageCaption: 'Cover photo by <a href="https://unsplash.com/@snowshade">Oleg Laptev</a> on <a href="https://unsplash.com/s/photos/megaphone">Unsplash</a>.'
+unlisted: true
```

Comment:
@dergigi let me know if we should add this text as normal blog post or have it unlisted and only make it accessible via direct link. We can also change this at any time.

Reply:
I'd opt for a direct link for now, i.e. `unlisted: true`